src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -323,8 +323,12 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
                 };
             }
 
-            // Add triggers to the payload
+            // Add HostId
             JObject result = new JObject();
+            string hostId = await _hostIdProvider.GetHostIdAsync(CancellationToken.None);
+            result.Add("hostId", hostId);
```

Comment:
Assuming the scale controller will use this, are there extra things we need to check, such as whether sync triggers runs correctly after a slot swap?

Reply:
Can you detail the scenario you're concerned about not working? ScaleController is already dependent on sync trigger updates being correct/timely after swaps, otherwise it'd be incorrectly monitoring/scaling.